Britannia/events/BTWK_storyline_events.txt

```diff
@@ -4,96 +4,676 @@
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Belgae ###################
+	option = { 
+		name = BTWK_story_event.0104.f
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_belgae
+					AND = {
+						is_primary_heir_of = title:k_belgae.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_belgae
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_belgae.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Cynwidion ###################
+	option = { 
+		name = BTWK_story_event.0104.g
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_cynwidion
+					AND = {
+						is_primary_heir_of = title:k_cynwidion.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_cynwidion
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_cynwidion.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Demetia ###################
+	option = { 
+		name = BTWK_story_event.0104.h
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_demetia
+					AND = {
+						is_primary_heir_of = title:k_demetia.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_demetia
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_demetia.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
```

Comment:
Don't remember if we discussed this, but are we going to have an option to choose Arthur? Feel like it should be brought up even if we don't actually add the option.

Reply:
I can add the option for Arthur, yes. Makes sense to give the player the option, but I'll make it so the AI has a 0% of choosing it - doesn't particularly make sense based on Uther's character/opinions. I can't remember either otherwise. Let me know what you think. If so, I'll probably localise it to make it seem like a "bad" option in Uther's eyes - just a subtle hint it isn't the right option.

Reply:
IIRC literally everyone else thought it was the obvious best choice, but Uther was too angry over Mordred Sr.'s death to consider it. So it depends on how much player agency you think there should be. If we do add it in I guess he vanilla way of dealing with it would be to give him a massive amount of stress...

Reply:
But even if we don't add it I was thinking we should still have the book event where Agricola (?) dares to mention him and Uther just completely disowns Arthur

Reply:
I thought about suggesting adding stress for the Arthur choice, but seemed a bit redundant as Uther was soon to die anyway. I suppose it wouldn't hurt though. I can add the Agricola event to the chain. Do you mind if I place the event out of sequence? I've moved event IDs around a few times so the events are chronological, but it's a bit of a pain to keep readjusting them when I have to slot in new events. Seeing as I'm labelling each event, it shouldn't hurt right? Up to you.